configure.py

```diff
@@ -1172,8 +1172,9 @@ def check_compiler(o):
   print_verbose(f"Detected {'clang ' if is_clang else ''}C++ compiler (CXX={CXX}) version: {version_str}")
   if not ok:
     warn(f'failed to autodetect C++ compiler version (CXX={CXX})')
-  elif clang_version < (8, 0, 0) if is_clang else gcc_version < (10, 1, 0):
-    warn(f'C++ compiler (CXX={CXX}, {version_str}) too old, need g++ 10.1.0 or clang++ 8.0.0')
+  elif clang_version < (8, 0, 0) if is_clang else gcc_version < (12, 2, 0):
+    warn(f'C++ compiler (CXX={CXX}, {version_str}) too old, need g++ 12.2.0 or clang++ 8.0.0')
+    exit(1)
```

Comment:
Is this expected?

Reply:
This is from https://github.com/nodejs/node/pull/54081/commits/bef819b88998595e723208c5d4081b7991bfaa51. The goal is to identify easily what needs to be changed in CI for this to land. The commit will be removed afterwards.